### PR TITLE
CODE RUB: Updated variable in retrieve all Attachments

### PR DESCRIPTION
**Pull request by deepshri7**

### CODE REVIEW

File: OtripleS.Web.Api.Tests.Unit/Services/Foundations/Attachments/AttachmentServiceTests.Exceptions.RetrieveAll.cs [hunk 1]
```diff
@@ -46,18 +50,22 @@ public void ShouldThrowDependencyExceptionOnRetrieveAllAttachmentsWhenSqlExcepti
         public void ShouldThrowServiceExceptionOnRetrieveAllAttachmentsWhenExceptionOccursAndLogIt()
         {
             // given
-            var exception = new Exception();
+            var Serviceexception = new Exception();
```
Comment (hassanhabib) [inline OtripleS.Web.Api.Tests.Unit/Services/Foundations/Attachments/AttachmentServiceTests.Exceptions.RetrieveAll.cs:53]: @deepshri7 naming here is also incorrect. `serviceException` is the correct choice.

File: OtripleS.Web.Api.Tests.Unit/Services/Foundations/Attachments/AttachmentServiceTests.Exceptions.RetrieveAll.cs [hunk 2]
```diff
@@ -25,13 +25,13 @@ public void ShouldThrowDependencyExceptionOnRetrieveAllAttachmentsWhenSqlExcepti
                 broker.SelectAllAttachments())
                     .Throws(sqlException);
 
-            // when
-            Action retrieveAllAttachmentsAction = () =>
-                this.attachmentService.RetrieveAllAttachments();
+            // When
```
Comment (ShriHumrudha) [inline OtripleS.Web.Api.Tests.Unit/Services/Foundations/Attachments/AttachmentServiceTests.Exceptions.RetrieveAll.cs:28]: small w

**Review by ShriHumrudha — CHANGES_REQUESTED** [review]

File: OtripleS.Web.Api.Tests.Unit/Services/Foundations/Attachments/AttachmentServiceTests.Exceptions.RetrieveAll.cs [hunk 3]
```diff
@@ -26,12 +26,12 @@ public void ShouldThrowDependencyExceptionOnRetrieveAllAttachmentsWhenSqlExcepti
                     .Throws(sqlException);
 
             // when
-            Action retrieveAllAttachmentsAction = () =>
-                this.attachmentService.RetrieveAllAttachments();
+             Action retrieveAllAttachmentAction = () =>
```
Comment (ShriHumrudha) [inline OtripleS.Web.Api.Tests.Unit/Services/Foundations/Attachments/AttachmentServiceTests.Exceptions.RetrieveAll.cs:29]: Why space?

Comment (deepshri7) [reply OtripleS.Web.Api.Tests.Unit/Services/Foundations/Attachments/AttachmentServiceTests.Exceptions.RetrieveAll.cs:29]: sorry but which space ?

Comment (ShriHumrudha) [reply OtripleS.Web.Api.Tests.Unit/Services/Foundations/Attachments/AttachmentServiceTests.Exceptions.RetrieveAll.cs]: All cleared now.

Comment (deepshri7) [reply OtripleS.Web.Api.Tests.Unit/Services/Foundations/Attachments/AttachmentServiceTests.Exceptions.RetrieveAll.cs]: still it saying changes requested.

Comment (deepshri7) [reply OtripleS.Web.Api.Tests.Unit/Services/Foundations/Attachments/AttachmentServiceTests.Exceptions.RetrieveAll.cs]: can u suggest me for this??

Comment (ShriHumrudha) [reply OtripleS.Web.Api.Tests.Unit/Services/Foundations/Attachments/AttachmentServiceTests.Exceptions.RetrieveAll.cs]: Build is failing. Merge the latest master.

Comment (deepshri7) [reply OtripleS.Web.Api.Tests.Unit/Services/Foundations/Attachments/AttachmentServiceTests.Exceptions.RetrieveAll.cs]: Not working

**Comment by ShriHumrudha** [comment]
There are 0 files changed.

**Pull request closed without merging**